src/Catalog.API/Extensions/Extensions.cs

```diff
@@ -37,12 +37,12 @@ public static void AddApplicationServices(this IHostApplicationBuilder builder)
         builder.Services.AddOptions<CatalogOptions>()
             .BindConfiguration(nameof(CatalogOptions));
 
-        if (builder.Configuration["AI:Ollama:Endpoint"] is string ollamaEndpoint && !string.IsNullOrWhiteSpace(ollamaEndpoint))
+        if (builder.Configuration["OllamaEnabled"] is string ollamaEnabled && bool.Parse(ollamaEnabled))
```

Comment:
Since we've changed what configuration values get read here, can we also update the appsettings.json files for these changes?

https://github.com/dotnet/eShop/blob/633dd1a6525e705a85ed3f65e5167c2f901e51a7/src/WebApp/appsettings.json#L13-L20

https://github.com/dotnet/eShop/blob/633dd1a6525e705a85ed3f65e5167c2f901e51a7/src/Catalog.API/appsettings.json#L27-L34

Reply:
ah yep, didn't think to dig into the appsettings to look for things that needed cleaning up